src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -915,9 +915,146 @@
   }
 
   /*
-        The following classes are used by the renderer when performing internal operations.
-        Using the browser calculations improves rendering performance.
-    */
+   * The following are styles for Detail Pane
+   */
+  .datagrid-outer-wrapper {
+    display: flex;
+    width: 100%;
+    overflow: hidden;
+  }
+
+  .datagrid-detail-open {
+    flex: 1 0 0;
+
+    .datagrid {
+      border-top-right-radius: 0;
+    }
+    .datagrid-footer {
+      border-bottom-right-radius: 0;
+    }
+    .pagination {
+      width: 100%;
+    }
+    .pagination-description-compact {
+      text-align: left;
+      flex: 1;
+    }
+    .datagrid-footer .pagination-list {
+      margin-right: 0;
+    }
+  }
+
+  .datagrid-wrapper {
+    width: 100%;
+
+    &.datagrid-detail-open {
+      width: auto;
+    }
+  }
+
+  .datagrid-row-detail-open {
+    position: relative;
+
+    &:before {
```

Comment:
I'm seeing a line on the RHS of the inset triangle when detail is opened. This seperates the row from the detail pane. Is that correct?

Reply:
Yeah, this needs to be different.